### ISSUE TITLE
release: v20.1.0-beta.5

### ISSUE THREAD

**Issue by otan** [cockroachdb/cockroach#46751]
Candidate SHA: WIP EDIT ME WIP.
Deployment status: WIP EDIT ME WIP.
Nightly Suite: WIP EDIT ME WIP.
 
[Release process checklist](https://cockroachlabs.atlassian.net/wiki/spaces/ENG/pages/73105625/Release+process)
 
Prep date: `WIP fill in date to roll out: usually Monday, a week before the release WIP`
- [ ] [Pick a SHA](https://cockroachlabs.atlassian.net/wiki/spaces/ENG/pages/73105625/Release+process#Releaseprocess-PickingaSHA)
  - [ ] fill in `Candidate SHA` above
  - [ ] notify #release-announce
  - [ ] email thread on releases@
- [ ] [Tag the provisional SHA](https://cockroachlabs.atlassian.net/wiki/spaces/ENG/pages/73105625/Release+process#Releaseprocess-TagtheprovisionalSHA)
- [ ] [Publish provisional binaries](https://cockroachlabs.atlassian.net/wiki/spaces/ENG/pages/73105625/Release+process#Releaseprocess-Publishtheprovisionalbinaries)
- [ ] Ack security@ on the generated Stackdriver Alert to confirm these writes were part of a planned release
 
Release Qualification
- [ ] [Check binaries](https://cockroachlabs.atlassian.net/wiki/spaces/ENG/pages/73105625/Release+process#Releaseprocess-Checkbinaries)
- [ ] [Deploy to test clusters](https://cockroachlabs.atlassian.net/wiki/spaces/ENG/pages/73105625/Release+process#Releaseprocess-DeploytoTestClusters)
- [ ] [Verify node crash reports](https://cockroachlabs.atlassian.net/wiki/spaces/ENG/pages/73105625/Release+process#Releaseprocess-Verifynodecrashreportsappearinsentry.io)
- [ ] [Start release qualification suite](https://cockroachlabs.atlassian.net/wiki/spaces/ENG/pages/73105625/Release+process#Releaseprocess-RuntheSuiteofNightlyRoachtests)
- [ ] [Start nightly suite](https://cockroachlabs.atlassian.net/wiki/spaces/ENG/pages/73105625/Release+process#Releaseprocess-RuntheSuiteofNightlyRoachtests)
- [ ] fill in `Deployment status` above with clusters and `Nightly Suite` with the link to Nightly TeamCity Job
 
One day after prep date:
- [ ] [Get signoff on roachtest failures](https://cockroachlabs.atlassian.net/wiki/spaces/ENG/pages/73105625/Release+process#Releaseprocess-RoachtestFailureSignoff)
 
- [Keep an eye on clusters](https://cockroachlabs.atlassian.net/wiki/spaces/ENG/pages/328859690/Release+Qualification#Monitoring-the-Qualification-Clusters until release date. Do not proceed below until the release date.
 
Release date: `WIP fill in date: usually Monday WIP`
- [ ] [Check cluster status](https://cockroachlabs.atlassian.net/wiki/spaces/ENG/pages/328859690/Release+Qualification#Monitoring-the-Qualification-Clusters)
- [ ] [Tag release](https://cockroachlabs.atlassian.net/wiki/spaces/ENG/pages/73105625/Release+process#Releaseprocess-Tagtherelease)
- [ ] [Bless provisional binaries](https://cockroachlabs.atlassian.net/wiki/spaces/ENG/pages/73105625/Release+process#Releaseprocess-Blesstheprovisionalbinaries)
 
- For production or stable releases in the latest major release series
  - [ ] Update the [Homebrew Formula](https://github.com/cockroachdb/homebrew-tap/blob/master/Formula/cockroach.rb)
  - [ ] Update the image tag in our [Orchestrator configurations](https://wiki.crdb.io/wiki/spaces/CRDB/pages/73105625/Release+process#Releaseprocess-Orchestratorconfigurations)
  - [ ] Update the image tag in Helm's [Orchestrator configurations](https://wiki.crdb.io/wiki/spaces/CRDB/pages/73105625/Release+process#Releaseprocess-Orchestratorconfigurations)
  - [ ] Announce version to the [registration cluster](https://cockroachlabs.atlassian.net/wiki/spaces/ENG/pages/73105625/Release+process#Releaseprocess-AnnounceVersionToRegCluster)
 
- [ ] Update docs
- [ ] External communications for release
- [ ] Clean up provisional tag from repository

**Comment by otan** [comment]
As per https://github.com/cockroachdb/cockroach/issues/45599, we are currently blocked on:
- [ ] #46625
- [ ] #46460 
- [ ] #43273
- [ ] #46152

**Comment by nvanbenschoten** [comment]
Fixes for #46625 (fixed by: https://github.com/cockroachdb/cockroach/pull/46744) and #43273 (fixed by: https://github.com/cockroachdb/cockroach/pull/46858) have been merged into `release-20.1`. A fix for #46460 (https://github.com/cockroachdb/cockroach/pull/46830) has an LGTM and should merge in the next hour.

I'm unsure about the fourth blocker: https://github.com/cockroachdb/cockroach/issues/46152. @lucy-zhang do you know its current status.

**Comment by lucy-zhang** [comment]
#46152 is waiting on #46504, which needs some test flakes de-flaked (as described in the comments) and another round of review. @pbardea and I are making some progress on de-flaking, and I can follow up with reviewers. We're making decent progress but this will probably have to go out tomorrow.

**Comment by nvanbenschoten** [comment]
The fix for #46460 just landed on `release-20.1`.

**Comment by otan** [comment]
Closing issue as we're aborting for RC.
